src/structures/Guild.js

```diff
@@ -207,12 +228,43 @@ class Guild extends Base {
     this.joinedTimestamp = data.joined_at ? new Date(data.joined_at).getTime() : this.joinedTimestamp;
 
     /**
-     * The value set for a guild's default message notifications
+     * The value set for the guild's default message notifications
      * @type {DefaultMessageNotifications|number}
      */
     this.defaultMessageNotifications = DefaultMessageNotifications[data.default_message_notifications] ||
       data.default_message_notifications;
 
+    /**
+     * The maximum amount of members the guild can have
+     * @type {?number}
+     * @name Guild#maximumMembers
+     */
+    if (typeof data.max_members !== 'undefined') this.maximumMembers = data.max_members;
+
+    /**
+     * The maximum amount of presences the guild can have
+     * @type {number}
+     */
+    this.maximumPresences = data.max_presences || this.maximumPresences || 5000;
```

Comment:
I did it this way so you always have the value, either as what the data provides, if any, the current max presences, if any, or 5000 which for 9/10 guilds, it's the default. I can make it be the same as the other code, but I feel like this is ok as is. If someone else wants me to change, I will 👍

Reply:
I see where your coming from 👌, but I feel like it would be better to not declare this property at all instead of somewhat guessing it based on the majority. The "default" on the `max_members` field is `250000` which is also prolly for 9/10 guilds as well so shouldn't that also be reflected in the same manner? Perhaps a little info element could be added to the jsdoc stating that this may not be fully accurate unless fetched?

Reply:
I guess I could make both max_presences and max_members optional, and make give it a `<warn></warn>` that the property must be fetched by calling Guild#fetch

Reply:
I'm not a fan of defaulting values to something that is true most of the time and not explicitly documented. I would prefer to see this nullable.